src/UserSpaceInstrumentation/InstrumentProcess.cpp

```diff
@@ -276,15 +278,18 @@ ErrorMessageOr<absl::flat_hash_set<uint64_t>> InstrumentedProcess::InstrumentFun
       }
       const TrampolineData& trampoline_data = it->second;
 
-      auto result = InstrumentFunction(pid_, function_address, function_id,
-                                       trampoline_data.address_after_prologue,
-                                       trampoline_data.trampoline_address);
-      if (result.has_error()) {
-        ERROR("Unable to instrument \"%s\": %s", function.function_name(),
-              result.error().message());
+      auto result_or_error = InstrumentFunction(pid_, function_address, function_id,
```

Comment:
nit. same here about using error_or[x] for consistency with returned type of the form error_or<type>